lmnet/lmnet/datasets/bdd100k.py

```diff
@@ -74,41 +92,58 @@ def __init__(self,
         self.num_workers = num_workers
 
         subset_dir = "train" if subset == "train" else "val"
-        self.img_dir = join(self.data_dir, "images", "100k", subset_dir)
-        self.anno_dir = join(self.data_dir, "labels", "100k", subset_dir)
+        self.img_dir = os.path.join(self.data_dir, "images", "100k", subset_dir)
+        self.anno_dir = os.path.join(self.data_dir, "labels", "bdd100k_labels_images_" + subset_dir + ".json")
         self.paths = []
         self.bboxs = []
 
         self._init_files_and_annotations()
 
     def _init_files_and_annotations(self):
-        img_paths = glob.glob(join(self.img_dir, "*.jpg"))
-        anno_paths = [join(self.anno_dir, splitext(basename(f))[0] + ".json")
-                      for f in img_paths]
-
-        self.paths = img_paths
-        for f in anno_paths:
-            with open(f) as fp:
-                raw = json.load(fp)
-            objs = raw["frames"][0]["objects"]
+        img_paths = dict([(os.path.basename(path), path)
+                          for path in glob.glob(os.path.join(self.img_dir, "*.jpg"))])
+        img_names = set(img_paths.keys())
+
+        anno_data = json.load(open(self.anno_dir))
+
+        counts = 0
```

Comment:
This variable looks better to remove.

Reply:
If you meant `counts` I can not remove it as it gives error `Variable not defined` but if you meant `max_boxes` I've removed that.

Reply:
In my understanding, the variable `counts` is only used in line 110 and line 114, it never return the information to the outside of this method, hence it could be removed.

Reply:
I'm sorry. It is a shame I missed that. It is removed.